pages/embed/[geoId]/[sectionId]/[chartId].js

```diff
@@ -86,30 +95,21 @@ function Embed({
           title: 'Embed code for this chart',
           subtitle:
             'Copy the code below, then paste into your own CMS or HTML. Embedded charts are responsive to your page width, and have been tested in Firefox, Safari, Chrome, and Edge.',
-          code: `<iframe src="https://dev.takwimu.africa/embed/${geoId}/${sectionId}/${chartId}" />`
+          code: `<iframe src="https://dev.dominion.africa/embed/${geoId}/${sectionId}/${chartId}" />`
         }}
       >
         {!chartData.isLoading &&
           chart.visuals.map(visual => (
             <ChartFactory
-              visual={visual}
+              key={visual.id}
+              definition={visual}
               profiles={profiles}
-              data={chartData.profileVisualsData}
+              data={chartData.profileVisualsData[visual.queryAlias].nodes}
```

Comment:
You forgot to pass `referenceData` if it exists 

```suggestion
              data={chartData.profileVisualsData[visual.queryAlias].nodes}
              referenceData={(() => {
                        const temp = chartData.profileVisualsData[`${visual.queryAlias}Reference`];
                       return temp ? temp.nodes : [];
              })()}
```